### DISCUSSION TITLE
New Time Picker component

### DISCUSSION THREAD

**Discussion by Dtsiantaris** [quasarframework/quasar#16771] [Ideas / Proposals]
Hello guys! I have created a really basic TimePicker. You can see a demo here:

![timepicker_demo](https://github.com/quasarframework/quasar/assets/44876922/1256ba31-b59c-4378-b911-aa8b6b8a1ae3)

I've been curious on why there isn't such a component already because i am sure people would love to use this in some scenarios instead of the existing QTime component. I have a repo [here](https://github.com/Dtsiantaris/Quasar-Custom-Time-Picker) where I provide the source code for what I implemented.

Sadly this is not an abstract component and its props, format etc are implemented in a way that the project i was currently working on needed. I would be happy to help implement this for the actual quasar project or if any of you guys find my idea useful to implement it yourselves.

Cheers! ✌️

**Comment by IosifPuha** [comment]
Just found this! You are indeed correct, people would like something like this but Quasar framework does not provide it. I was looking for something similar to the picker in iOS timer for my application, your idea might be very useful!

**Comment by Inklikdevteam** [comment]
Thanks for this. My client asked me that his staff hates that clock to select time. They always mess it up as they often forgot to select the PM.

**Comment by amplexdenmark** [comment]
This looks great, much nicer than the standard Quasar component.
I am tempted to use this, but you have not provided a license in your repo.
